architecture/additional_concepts/dynamic_admission_controllers.adoc

```diff
@@ -0,0 +1,535 @@
+<4> Information about how to connect to, trust, and send data to the webhook server.
+<5> The project where the front-end service is created.
+<6> The name of the front-end service.
+<7> The webhook URL used for admission requests.
+<9> A PEM-encoded CA certificate that signs the server certificate used by the webhook server.
+<10> Rules that define when the API server should use this controller.
+<11> The operation(s) that triggers the API server to call this controller: 
+* create
+* update
+* delete
+* connect
+<10> Specifies how the policy should proceed if the webhook admission server is unavailable. 
+Either `Ignore` (allow/fail open) or `Fail` (block/fail closed). 
+
+[[admission-webhooks-v]]
+*Validating admission webhooks* are invoked during the validation phase of the admission process. 
+This phase allows the enforcement of invariants on particular API resources 
+to ensure that the resource does not change again. The Pod Node Selector is also an example of a validation admission, 
+by ensuring that all `nodeSelector` fields are constrained by the node selector restrictions on the project. 
+
+[[architecture-additional-concepts-dynamic-admission-webhooks-ex-v]]
+//http://blog.kubernetes.io/2018/01/extensible-admission-is-beta.html
+.Sample validating admission webhook configuration:
+
+[source,yaml]
+----
+apiVersion: admissionregistration.k8s.io/v1beta1
+  kind: ValidatingWebhookConfiguration <1>
+  metadata:
+    name: <controller_name> <2>
+  webhooks:
+  - name: <webhook_name> <3>
+    clientConfig: <4>
+      service:
+        namespace: default  <5>
+        name: kubernetes <6>
+       path: <webhook_url> <7>
+      caBundle: <cert> <8>
+    rules: <9>
+    - operations: <10>
+      - <operation>
+      apiGroups:
+      - ""
+      apiVersions:
+      - "*"
+      resources:
+      - <resource>
+    failurePolicy: <policy> <11>
+----
+
+<1> Specifies a validating admission webhook configuration.
+<2> The name for the webhook admission object.
+<3> The name of the webhook to call.
+<4> Information about how to connect to, trust, and send data to the webhook server.
+<5> The project where the front-end service is created.
+<6> The name of the front-end service.
+<7> The webhook URL used for admission requests.
+<8> A PEM-encoded CA certificate that signs the server certificate used by the webhook server.
+<9> Rules that define when the API server should use this controller.
+<10> The operation that triggers the API server to call this controller. 
+* create
+* update
+* delete
+* connect
+<11> Specifies how the policy should proceed if the webhook admission server is unavailable. 
+Either `Ignore` (allow/fail open) or `Fail` (block/fail closed). 
+
+[NOTE]
+====
+Fail open can result in unpredictable behavior for all clients.
+====
+ 
+[[architecture-additional-concepts-dynamic-admission-webhooks-writing]]
+=== Create the Admission Webhook
+
+First deploy the external webhook server and ensure
```

Comment:
First,
Maybe this should be formatted as the 1st step. If so, you can probably drop "First" altogether.